### DISCUSSION TITLE
template with url

### DISCUSSION THREAD

**Discussion by dodyg** [OrchardCMS/OrchardCore#8433] [Ideas]
I am thinking out loud here.

I think it should be easier to create dynamic pages that are just consisted of templates. Right now AFAIK you have to create a minimal content type and then deal with the template separately.

Something in the form of

- Title
- Template Editor
- Url

**Comment by dodyg** [comment]
Especially now with the Monaco Editor being introduced https://github.com/OrchardCMS/OrchardCore/pull/8266. Hopefully we can have monaco editor as an editor in a content part.

**Comment by dodyg** [comment]
I wonder if 

```{{ Model.ContentItem.Content.Paragraph.Content.Html | liquid }}``` is enough to execute full blown liquid programming

**Comment by deanmarcussen** [comment]
I'm trying to understand how this is not just a `LiquidPart` ? 

`{{ Model.ContentItem.Content.Paragraph.Content.Html | liquid }}` yes, but obviously that won't trigger a template, which I think is what you are wanting to do.

**Reply by dodyg** [reply]
The liquid part doesn't stand alone - you can only do one type of template per content type.

![liquid-part](https://user-images.githubusercontent.com/1761249/105852795-95337f00-5fed-11eb-8574-eb2b5f1e1ef7.png)

I am talking about a content type which purpose is to just  hold liquid template and execute them. 

So a content type with the following definition:

1. Title
2. Liquid Template
3. Url

So now you can create pages such as

1. list-of-all-of-our-blogs
2. our-media-list

Think of it as a programmable page.

**Reply by deanmarcussen** [reply]
I still think this is a `LiquidPart` 
Make a ContentType, add the `LiquidPart` to it. In the template for the content type just only render the liquid part.
